Pistachio 4.4

abhijit-mahajani-imgtec opened the pull request:
Merge dev-pistachio-4.4.7 branch into master-pistachio to make default kernel as 4.4.7
also increased version number as 0.10.2 to mark as Kernel 4.4 based releases.

But this is to be merged only after #145 is merged. lets first have both branches in sync and in next release we can make default master-pistachio as 4.4

mayank-sirotiya-imgtec committed:
pistachio: Update kernel version to 4.4

Signed-off-by: Mayank Sirotiya <Mayank.Sirotiya@imgtec.com>

mayank-sirotiya-imgtec committed:
pistachio: add kernel config and patches for 4.4

Signed-off-by: Mayank Sirotiya <Mayank.Sirotiya@imgtec.com>
Signed-off-by: Shraddha Chaudhari <Shraddha.Chaudhari@imgtec.com>

shraddha-chaudhari-imgtec committed:
Merge branch 'master' into dev-pistachio-4.4.7

Signed-off-by: Shraddha Chaudhari <Shraddha.Chaudhari@imgtec.com>

Conflicts:
	include/subdir.mk

shraddha-chaudhari-imgtec committed:
pistachio: fix patches for kernel version 4.1.23

Signed-off-by: Shraddha Chaudhari <Shraddha.Chaudhari@imgtec.com>

Commit:
Merge pull request #111 from shraddha-chaudhari-imgtec/dev-pistachio-4.4.x

Dev pistachio 4.4.x

Commit:
Merge remote-tracking branch 'upstream/master-pistachio' into dev-pistachio-4.4.7

nikhil-zinjurde-imgtec committed:
Merge pull request #121 from abhijit-mahajani-imgtec/dev-pistachio-4.4.7

Merge upstream/master-pistachio into Dev pistachio 4.4.7

Commit:
patches-4.4: add kernel patch to fix dma issue in spi-img-spfi driver

Signed-off-by: Abhijit Mahajani <Abhijit.Mahajani@imgtec.com>

Commit:
Merge remote-tracking branch 'upstream/master-pistachio' into 4.4

nikhil-zinjurde-imgtec committed:
Merge pull request #135 from abhijit-mahajani-imgtec/dev-pistachio-4.4.7

Dev pistachio 4.4.7:add kernel patch to fix dma issue in spi-img-spfi driver

nikhil-zinjurde-imgtec committed:
creator-platform configs: Update version numbers to 0.10.1 for 4.4.x kernel version

Signed-off-by: Nikhil Zinjurde <nikhil.zinjurde@imgtec.com>

Commit:
Merge remote-tracking branch 'upstream/dev-pistachio-4.4.7' into pistachio

Commit:
Bump version number to 0.10.1 for kernel 4.4 release

Signed-off-by: Abhijit Mahajani <Abhijit.Mahajani@imgtec.com>

abhijit-mahajani-imgtec commented:
test with ca8210 profile

nikhil-zinjurde-imgtec committed:
Merge pull request #142 from nikhil-zinjurde-imgtec/dev-pistachio-4.4.7

creator-platform configs: Update version numbers to 0.10.1 for 4.4.x …

Commit:
Merge remote-tracking branch 'upstream/master-pistachio' into dev-pistachio-4.4.7

Signed-off-by: Abhijit Mahajani <Abhijit.Mahajani@imgtec.com>

Commit:
Merge branch 'master-pistachio' of https://github.com/abhijit-mahajani-imgtec/openwrt into pistachio-4.4

Signed-off-by: Abhijit Mahajani <Abhijit.Mahajani@imgtec.com>

nikhil-zinjurde-imgtec committed:
Merge pull request #145 from abhijit-mahajani-imgtec/update_dev_pistachio_4.4.7

Update dev pistachio 4.4.7

nikhil-zinjurde-imgtec commented:
test with jenkins - dev

nikhil-zinjurde-imgtec commented:
The image with these PR changes seems to work reasonably well, 6lowpan works fine and also the available board tests.  Able to connect to Wi-Fi from LuCI, but the wlan interface does not get an IP.  There is also no route set for wlan0.

    root@OpenWrt:/# route
    Kernel IP routing table
    Destination     Gateway         Genmask         Flags Metric Ref    Use Iface
    default         192.168.92.254  0.0.0.0         UG    0      0        0 eth0
    10.10.10.0      *               255.255.255.0   U     0      0        0 wlan0-1
    192.168.92.0    *               255.255.255.0   U     0      0        0 eth0
    192.168.92.254  *               255.255.255.255 UH    0      0        0 eth0

francois-berder committed:
update uccp420wlan driver to version 6.8

Signed-off-by: Francois Berder <Francois.Berder@imgtec.com>

francois-berder committed:
update firmware of uccp420wlan driver to version 6.8

Signed-off-by: Francois Berder <Francois.Berder@imgtec.com>

Commit:
Merge branch 'master-pistachio' of https://github.com/abhijit-mahajani-imgtec/openwrt into pistachio-4.4

Commit:
Merge remote-tracking branch 'upstream/master-pistachio' into dev-pistachio-4.4.7

Commit:
Merge remote-tracking branch 'upstream/master-pistachio' into pistachio-4.4

Commit:
Merge remote-tracking branch 'upstream/master-pistachio' into dev-pistachio-4.4.7

Commit:
pistachio-4.4: add CRDA database patch

The default CRDA database is empty, replacing it with
a copy from wireless-regdb repository.

Signed-off-by: Francois Berder <Francois.Berder@imgtec.com>
Signed-off-by: Abhijit Mahajani <Abhijit.Mahajani@imgtec.com>

Commit:
modules.mk: configure cfg80211 to use CRDA support.

This option CONFIG_CFG80211_CRDA_SUPPORT seems to have added in kernel 4.4,
it was not present in kernel 4.1.

Signed-off-by: Abhijit Mahajani <Abhijit.Mahajani@imgtec.com>

nikhil-zinjurde-imgtec committed:
Merge pull request #155 from abhijit-mahajani-imgtec/dev-pistachio-4.4.7

Update Dev pistachio 4.4.7

nikhil-zinjurde-imgtec committed:
Merge pull request #149 from francois-berder-imgtec/update-wifi-driver

Update wifi driver to version 6.8

Commit:
pistachio: update license header

Update Copyright notice from BSD-3C to standard OpenWrt GPLv2

Signed-off-by: Abhijit Mahajani <Abhijit.Mahajani@imgtec.com>

Commit:
uboot-envtools: add pistachio support

Remove 30_uboot-envtools from target/linux/pistachio to proper location.

Signed-off-by: Abhijit Mahajani <Abhijit.Mahajani@imgtec.com>

nikhil-zinjurde-imgtec committed:
Merge pull request #161 from abhijit-mahajani-imgtec/update_license

pistachio: update license header

francois-berder committed:
Add missing patch from internal repository to fix mac80211

File 127-FROMLIST-mac80211-minstrel-Enable-STBC-and-LDPC-for-.patch
introduce a new variable which is not initialised. This caused
issues with some wirelessrouters and we could not get an IP
address although we could associate with the network.

Signed-off-by: Francois Berder <Francois.Berder@imgtec.com>

nikhil-zinjurde-imgtec committed:
Merge pull request #162 from abhijit-mahajani-imgtec/uboot-envtools

uboot-envtools: add pistachio support

nikhil-zinjurde-imgtec committed:
Merge pull request #164 from francois-berder-imgtec/fix-ip-issue

Fix IP address issue if using linux kernel 4.4.7

Commit:
Merge branch 'master-pistachio' of https://github.com/abhijit-mahajani-imgtec/openwrt into pistachio-4.4

Signed-off-by: Abhijit Mahajani <Abhijit.Mahajani@imgtec.com>

Commit:
Merge remote-tracking branch 'upstream/dev-pistachio-4.4.7' into pistachio-4.4

Signed-off-by: Abhijit Mahajani <Abhijit.Mahajani@imgtec.com>
